### PR TITLE
add snippet try else

### PR DESCRIPTION
**Pull request by lazargugleta**
## What does your PR belong to?
- [x] Snippets

### CODE REVIEW

File: snippets/most_frequent.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: most_frequent
+tags: array,beginner
+---
+
+Method return the most frequent element that appears in a `list`.
```
Comment (Chalarangelo) [inline snippets/most_frequent.md]: Please rephrase to: 
```suggestion
Returns the most frequent element in a list.
```

File: snippets/most_frequent.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Upon calling the function, in return you a `value`, which most frequently appears in the `list`.
```
Comment (Chalarangelo) [inline snippets/most_frequent.md]: Please rephrase to:
```suggestion
Use `set(list)` to get the unique values in the `list` in combination max()` to find the element that has the most appearances.
```

File: snippets/most_frequent.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+```py
+def most_frequent(list):
+    return max(set(list), key = list.count)
```
Comment (Chalarangelo) [inline snippets/most_frequent.md]: Please fix indentation to be 2 spaces, not 4.

**Comment by Chalarangelo** [comment]
(Btw I forgot, I think tags should be `list,beginner` instead of `array,beginner`. :sweat_smile:

**Comment by lazargugleta** [comment]
I think I fixed all the above. Check again just to be sure! @Chalarangelo

**Pull request merged**